src/host/renderData.cpp

```diff
@@ -177,9 +177,9 @@ COLORREF RenderData::GetCursorColor() const noexcept
 //  (the highest overlay should be given last)
 // Return Value:
 // - Iterable set of overlays
-const std::vector<RenderOverlay> RenderData::GetOverlays() const noexcept
+const std::vector<Microsoft::Console::Render::RenderOverlay> RenderData::GetOverlays() const noexcept

```

Comment:
I'd probably prefer that this file just includes a `using namespace Microsoft::Console::Render` but this is also fine like this

Reply:
Interested in the thinking here.  using namespace has so many downsides.   No answer needed for this PR.  This can be a future discussion.

Reply:
@zadjii-msft I think for consistency that makes sense. But my thinking there was that since the ```Microsoft::Console::Render``` was only used there it didn't hurt using the full qualifier.